python/cuml/tests/test_label_encoder.py

```diff
@@ -218,9 +218,11 @@ def test_inverse_transform_cupy_numpy(orig_label, ord_label,
 
     # test if inverse_transform is correct
     reverted = le.inverse_transform(ord_label)
+
+    reverted = cp.asarray(reverted)
+    expected_reverted = cp.asarray(expected_reverted)
```

Comment:
Should we just declare them as `cp.arrays` as part of the test parametrization directly then?

Reply:
Yes, you are right, the issue only appear when a cuDF DatFrame is indexed from a host array. A change in test parametrization alone is indeed sufficient.